tests/contenttypes_tests/tests.py

```diff
@@ -21,7 +21,7 @@
 from django.utils.encoding import force_str, force_text
 
 from .models import (
-    Article, Author, ModelWithNullFKToSite, Post, SchemeIncludedURL,
+    Answer, Article, Author, ModelWithNullFKToSite, Post, Question, SchemeIncludedURL,
```

Comment:
Make sure you're using the newest version of isort. The check passes on Jenkins because of https://github.com/timothycrosley/isort/issues/423.